scripts/trivia.js

```diff
@@ -784,11 +785,16 @@ TriviaGame.prototype.finalizeAnswers = function () {
                         if (this.triviaPlayers[id2].name === name) break;
                     }
                     if(trivData.eventFlag){
-                        eventElimPlayers.push(this.triviaPlayers[id2].name);
+                        sortArray.push(this.triviaPlayers[id2].name);
                     }
                     this.unjoin(id2);
                 }
             }
+            //Add players to sort array as eliminated, then randomize listing so that it isn't based on join order when ties happen.
+            sortArray.sort(function(a, b){return 0.5 - Math.random()});
```

Comment:
Probably. I tried doing that first, but I couldn't get it to work. It could have just been my syntax though. 

this.triviaPlayers[id].sort(function(a, b){return 0.5 - Math.random()});

or something to that effect is what I tried, but it caused an "is not a valid function" error.